src/core/providers/strategies/helpers/signTransactions/helpers/getCommonData/getCommonData.ts

```diff
@@ -16,11 +15,11 @@ import { getFeeData } from '../getFeeData';
 import { getUsdValue } from '../getUsdValue';
 import { getExtractTransactionsInfo } from './helpers/getExtractTransactionsInfo';
 import { getHighlight } from './helpers/getHighlight';
+import { getPpuOptions } from './helpers/getPpuOptions';
```

Comment:
Have an `index.ts` located in the `helpers` folder directly and export these helpers from there. Applicable to the helper below too.